openshift/postgresql.dc.json

```diff
@@ -59,12 +59,24 @@
             "value": "1Gi"
         },
         {
-            "description": "Version of PostgreSQL image to be used (9.2, 9.4, 9.5 or latest).",
+            "description": "Version of PostgreSQL image to be used (9.5 or 9.6).",
             "displayName": "Version of PostgreSQL Image",
             "name": "IMAGE_STREAM_VERSION",
             "required": true,
-            "value": "9.5"
+            "value": "9.6"
```

Comment:
this is the tag for the `IMAGE_STREAM_NAME` you are planning to use (postgresql-9.6-oracle-fdw), so it will need to be `v1-stable`.  You could also just always pass it in as a parameter but this default value of 9.6 won't work.

Reply:
thx for the catch.. I'm still a little bit confused as the Jenkinsfile seems to only use the IMAGE_STREAM_NAME environment variable in the *output* of the deployment (e.g. naming/tagging the deployed image).  I don't see how  changing this value would affect anything in the deployed container.  For example the IMAGE_STREAM_NAMESPACE is blank... how do any of the IMAGE_STREAM_* stuff affect the source of the postgresql.dc.json file??
/cc @stephenhillier

Reply:
I see `IMAGE_STREAM_NAME` set in Jenkinsfile (line 207):
```
"IMAGE_STREAM_NAME=gwells-postgresql-${DEV_SUFFIX}-${PR_NUM}",
```

It is consumed in the DeploymentConfig ImageChange trigger (postgresql.dc.json):

```
"triggers": [
                    {
                        "imageChangeParams": {
                            "automatic": true,
                            "containerNames": [
                                "postgresql"
                            ],
                            "from": {
                                "kind": "ImageStreamTag",
                                "name": "${IMAGE_STREAM_NAME}:${IMAGE_STREAM_VERSION}",
                                "namespace": "${IMAGE_STREAM_NAMESPACE}"
                            },
                            "lastTriggeredImage": ""
                        },
                        "type": "ImageChange"
                    },
```

It's a bit confusing but I think the trigger is used to override the image specified in the DeployConfig (which is `"image": " "`).  The docs say if the image in the ImageChange trigger differs from the deploy image, that causes the deployment to load the image in the trigger.  So basically, whatever you pass into `${IMAGE_STREAM_NAME}:${IMAGE_STREAM_VERSION},` will be deployed, but that needs to be a real image and tag.  The reason the comments say "9.6, 9.5, latest" is because the file was originally for the official OpenShift postgres image.

As for namespace="", the apply commands are being run in the context of the `DEV_PROJECT` namespace already so I think it just defaults to that, and then falls back on "openshift" (where some of the base images are kept)